core/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java

```diff
@@ -56,7 +56,18 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
         execute(terminal, arg, env);
     }
 
-    // pkg private for testing
+    /**
+    * Remove the plugin named {@code plugin named}.
+    *
+    * @param terminal       the Terminal to use for input/output
+    * @param pluginName     the name of the plugin to remove.
```

Comment:
Let's remove the period from the end of these, they are typically not sentences.